### ISSUE TITLE
Cannot run SteamVR with Oculus Quest

### ISSUE THREAD

**Issue by embedisbae** [JackD83/ALVR#188]
# Please follow the general troubleshooting steps first:

- [yes ] I ran "add_firewall_rules.bat" as adimistrator and checked that the ports are open in the firewall
- [yes ] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/JackD83/ALVR/wiki)

### Bug reports:

I tried connecting my Oculus Quest to ALVR and pressing Start Server immediatly crash
(The error is: Unhandled Exception.
ExceptionCode=C0000005
Address=00007FFED813C344 (00007FFEE0E30000 + FFFFFFFFF730C344)
#### Environement

What hardware do you have: CPU, GPU, Audio
CPU: Intel Core i7-6700HQ
GPU: NVIDIA GeForce GTX 960M
Audio: HDMI Audio
What version of ALVR and SteamVR have you tried.
ALVR: experimental 10
SteamVR: 1.12.2 (latest)
Do you have the latest Windows updates? Run "winver" in Windows 10 to get the version number
Version 1903 (Build 18362.778)

**Comment by MPfaffe** [comment]
as it is a notebook try to make sure it (ALVR, steamvr, VRserver) runs with the dedicated nvidia graphics card, maybe steamvr tries to use the integrated intel graphics.

hope it helps

Is this your first try to use steamvr with your notebook? because in my Opinion the mobile GPU GTX960m seems to be a bit underpowered. please don't expect to much for your VR experience with that GPU.

**Comment by clach04** [comment]
@embedisbae version https://github.com/JackD83/ALVR/releases/tag/v11-PRE works for me (and its already out of date :), a new version dropped a couple of days ago) on a desktop PC

**Comment by ratozumbi** [comment]
I am having the same issue with v11
I have the same graphics card + intel graphics card
I already used this same laptop to play The Lab using the Lenovo Explorer HMD, so I am confident the problem it's not lack of hardware horsepower 
I know I can force nvidia to be used on a specific configured program using nvidia panel, but ALVR is not showing up on the list for to add a configuration. Anyway, the global configuration is set to use the gtx 960m

**Comment by AndreyLebedenko** [comment]
One more note: as you know, GTX 960M does not support H.265 (HEVC), therefore, an attempt to start ALVR with this option selected in the "Video" tab will cause the application to stop with an error. Make sure you've selected H.264 and restarted the application before starting SteamVR.

**Comment by MPfaffe** [comment]
@ratozumbi 
isn't it possible to add the ALVR.exe to the list in the nvidia control panel/ 3d settings/ application settings?

i think @AndreyLebedenko is right. maybe also try to run it with 60 hz limit option in ALVR Server app

**Comment by zangief618** [comment]
I am having this same issue with my laptop with a GTX 2070 super.
ALVR is not in my list of programs to force the high performance card.
I am able to connect my Quest but as soon as the Steam home room appears, it is clear that the graphics are struggling.
When installing ALVR it noticed that I only had integrated graphics and did not find the GTX.
Figured I could change it as a setting once I got ALVR up and running.
Thanks in advance for any advice.
